frontend/src/models/annotationsModel.ts

```diff
@@ -1,24 +1,41 @@
-import { actions, afterMount, kea, listeners, path, reducers } from 'kea'
+import { actions, afterMount, connect, kea, listeners, path, reducers, selectors } from 'kea'
 import api from 'lib/api'
 import { deleteWithUndo } from 'lib/utils'
 import type { annotationsModelType } from './annotationsModelType'
-import { AnnotationType } from '~/types'
+import { AnnotationType, ParsedAnnotationType } from '~/types'
 import { loaders } from 'kea-loaders'
+import { teamLogic } from 'scenes/teamLogic'
+import { dayjs } from 'lib/dayjs'
 
 export type AnnotationData = Pick<AnnotationType, 'date_marker' | 'scope' | 'content' | 'dashboard_item'>
 export type AnnotationDataWithoutInsight = Omit<AnnotationData, 'dashboard_item'>
 
+export function parseAnnotation(annotation: AnnotationType, projectTimezone: string): ParsedAnnotationType {
```

Comment:
Not a big fan of the naming personally 😅 Don't have an immediate better suggestion though...

Reply:
Hmm, me too, I tried something else, see now.

Reply:
That's the word! Nice!